src/ui/react/src/adapter/alloy-editor.js

```diff
@@ -99,22 +101,24 @@
          * @method _renderUI
          */
         _renderUI: function() {
-            var editorUIElement = document.createElement('div');
-            editorUIElement.className = 'ae-ui';
+            if (!this._destroyed) {
```

Comment:
@ipeychev, can you think of a better way of doing this?

The thing is that because we call `_renderUI` asynchronously on `langResourcesLoaded`, it might still happen that we render the UI after the `destroy` method has been called.

The visible result is that, even if CKEditor does not complain now, we still get issues because of the `_mousedown` events we attached and couldn't remove because of `destructor` being invoked first.